packages/date/src/DateRange.js

```diff
@@ -301,7 +302,7 @@ const DateRange = ({
           {...datepickerProps}
           startDate={getDateValue(startValue)}
           startDateId={startId}
-          enableOutsideDays
```

Comment:
I'm hesitant about directly exposing more `react-dates` props to users since we're considering moving away from the library and would want to minimize any breaking changes when migrating. Can we move `enableOustideDays` above `{...datepickerProps}` so that your override works as intended?

This solves the issue but makes it more clear that it's not necessarily a supported prop or feature going forward, if we do decide to switch underlying libraries

Reply:
I am just wondering why that property was hardcoded to be that in the first place, maybe it was just an oversight. I had expected I could pass any of the available react-dates props into the dateRangePicker props and it be respected. My preference would let dateRangeProps provide speak for themselves and than any other custom properties be specific to the Availity implementation. For example, 'numberOfMonths', 'openDirection' etc are all specific to the react-dates and you cannot override them because of their position on the input. I would to put all of these above the spread of {...datepickerProps} but that might be overkill.

What library is being considered now for handling dates?

Reply:
I am assuming it was an oversight. Looks like we need to revisit how those props are spread. At a quick glance it seems like we can clean it up a bit. We can handle that in another pr though. From my understanding, it's not so much that we have a new lib in mind, but rather the `react-dates` package is not highly maintained anymore. We also want to switch away from a `moment` based lib.

Reply:
Yeah I think that was just an oversight. And like Jordan said, no particular library in mind at the moment, but for maintenance and accessibility reasons we may have to move away